### DISCUSSION TITLE
Build powerful automated workflows with the new n8n integration

### DISCUSSION THREAD

**Discussion by jamie-budibase** [Budibase/budibase#13113] [Announcements]
**Release Highlights:**

- Leverage n8n's extensive library of 775 nodes to unlock a range of powerful automated workflows directly within Budibase.
- Streamline your data administration by managing Budibase-stored data seamlessly through n8n.

Read the [launch blog](https://budibase.com/blog/updates/2024/n8n-integration/) to learn more about this release.

![make-work-flow-with-n8n](https://github.com/Budibase/budibase/assets/154342393/c4f8c526-37f6-4f0c-acb4-f434586b6205)

**Comment by sdsys-ch** [comment]
Can't wait to try this since n8n has been a workhorse for us in the past year. 

I've already set up automation between n8n, our budibase integrated postgres db and external APIs and this looks right up our alley. Great news!

Thank you!
